src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -112,10 +110,7 @@ const NavBar: React.FunctionComponent<PropsType> = ({ redirectionUrl }) => {
                 onClick={() => setIsProfileModalOpen(true)}
               >
                 <img
-                  src={
-                    users?.[user.uid]?.pictureUrl ||
-                    "/anonymous-profile-icon.jpeg"
-                  }
+                  src={profile.pictureUrl}
```

Comment:
same comment: if the user is logged in but has no pictureUrl, he needs to be able to click on this img to log out or edit his profile